src/overmap.cpp

```diff
@@ -2321,6 +2326,9 @@ void overmap::place_cities()
             tmp.s = size;
             cities.push_back( tmp );
 
+            DebugLog( D_INFO, DC_ALL ) << "CITY:[" << tmp.name << "] at {" << tmp.x << "," << tmp.x <<
```

Comment:
This spams the logs a LOT, it might be interesting but I don't think we need it happening by default.

Reply:
Right, it was debug feature I forgot to cut-off.